pulsar-broker/src/test/java/org/apache/pulsar/client/impl/ProducerCloseTest.java

```diff
@@ -99,6 +103,25 @@ public void testProducerCloseFailsPendingBatchWhenPreviousStateNotReadyCallback(
         }
     }
 
+    @Test(timeOut = 10_000)
+    public void brokerCloseTest() throws Exception {
+        initClient();
```

Comment:
The client will be created before executing the method, we don't need to init it again.
Not sure why all the tests in this class call initClient(), this is a useless call and the old client
will not be closed. We should remove all the `initClient()` calls for this method.

Reply:
OK